extensions/interactions/TextInput/TextInput.html

```diff
@@ -9,13 +24,14 @@
   <div class="row text-input-container">
     <form ng-submit="submitAnswer(answer)" role="form" class="form-horizontal">
       <div class="col-lg-12 col-md-12 col-sm-12">
-        <schema-based-editor schema="schema" local-value="answer" label-for-focus-target="labelForFocusTarget">
-        </schema-based-editor>
-      </div>
-      <div ng-show="rows > 1">
-        <button type="submit" class="btn btn-default" style="margin-top: 15px; margin-left: 16px;">
-          Submit
-        </button>
+        <input ng-hide="rows > 1" type="text" ng-model="answer" class="text-input-single-line-field form-control" placeholder="{{placeholder}}" focus-on="<[labelForFocusTarget]>">
+          <div ng-show="rows > 1">
+            <schema-based-editor schema="schema" local-value="answer" label-for-focus-target="labelForFocusTarget">
+            </schema-based-editor>
+            <button type="submit" class="btn btn-default" style="margin-top: 15px; margin-left: 16px;">
```

Comment:
Oops, sorry -- one other thing I noticed: try creating a textarea here by setting the number of rows to, say, 4. In this case, it looks like the 'submit' button is misaligned; it's too far to the right.

Could we adjust (or remove) the margin-left here in order to align them?

![misaligned](https://cloud.githubusercontent.com/assets/10575562/10058995/3b191fdc-61fd-11e5-9844-735f9eaf850c.png)